lib/after-prepare.js

```diff
@@ -63,3 +64,19 @@ module.exports = function (logger, platformsData, projectData, hookArgs) {
 
 	return Promise.all(transformations);
 }
+
+function getPlatformData($injector, $projectData, platform) {
+	let platformData;
+	try {
+		// used in CLI 6+
+		const platformsDataService = $injector.resolve("platformsDataService");
+		platformData = platformsDataService.getPlatformData(platform, $projectData);
+	} catch (err) {
+		// Used in CLI 5.4.x and below:
+		const platformsData = $injector.resolve("platformsData");
+		platformData = platformsData.getPlatformData(platform, $projectData);
+
```

Comment:
why is this new line ?

Reply:
https://github.com/rosen-vladimirov/nativescript-localize/commit/c82e678fefea3413bfaac9ae0def58a3faaea581#diff-cc3d3cdb65af528050edd09a33ed47faR48